src/core/lib/surface/call.h

```diff
@@ -265,6 +265,10 @@ void grpc_call_log_batch(const char* file, int line, const grpc_op* ops,
 
 void grpc_call_tracer_set(grpc_call* call, grpc_core::ClientCallTracer* tracer);
 
+// Both sets and manages call tracer life cycle using ClientCallTracerWrapper.
```

Comment:
Please specifically document the lifecycle for the ClientCallTracer, not referencing ClientCallTracerWrapper (which is an implementation detail).